doc/qat/USAGE.md

```diff
@@ -22,72 +22,99 @@ When deploying custom-built resource driver image, change `image:` lines in
 
 ## `deployment/` directory contains all required YAMLs:
 
-* `deployments/qat/device-class.yaml` - pre-defined DeviceClass that ResourceClaims can refer to.
+* `deployments/qat/resource-class.yaml` - pre-defined DeviceClass that ResourceClaims can refer to.
```

Comment:
device-class.yaml is correct, the git tree had old name of the file, see #23

Reply:
Thank you! :)